lib/src/mock_user.dart

```diff
@@ -74,18 +80,27 @@ class MockUser with EquatableMixin implements User {
   @override
   String? get refreshToken => _refreshToken;
 
+  set idTokenAuthTime(DateTime time) {
+    _idTokenAuthTime = time;
+  }
+
+  set idTokenExp(DateTime time) {
+    _idTokenExp = time;
+  }
+
   @override
   Future<String> getIdToken([bool forceRefresh = false]) {
-    var payload = {
+    final payload = {
       'name': displayName,
       'picture': photoURL,
       'iss': 'fake_iss',
       'aud': 'fake_aud',
-      'auth_time': 1655946582,
+      'auth_time': _idTokenAuthTime.millisecondsSinceEpoch ~/ 1000,
       'user_id': uid,
       'sub': uid,
-      'iat': 1656302136,
-      'exp': 1656305736,
+      'exp': (_idTokenExp ?? DateTime.now().add(Duration(hours: 1)))
+              .millisecondsSinceEpoch ~/
+          1000,
```

Comment:
I set exp's default value to one hours late. I reference this document: https://firebase.google.com/docs/reference/admin/node/firebase-admin.auth.decodedidtoken

Reply:
Thanks for the explanation. Would you mind adding this link as a comment, so that people can easily refer to it later?

Reply:
Done